accessories/TemperatureSensor.py

```diff
@@ -18,6 +16,6 @@ def __init__(self, *args, **kwargs):
         serv_temp = self.add_preload_service('TemperatureSensor')
         self.char_temp = serv_temp.configure_char('CurrentTemperature')
 
-    @AsyncAccessory.run_at_interval(3)
-    async def run(self):
+    @Accessory.run_at_interval(3)
+    def run(self):
```

Comment:
Why not keep it `await def run`?

Reply:
`char.set_value` is still be blocking, so this should not be executed in the event loop.